integrations/oap/kubernetes/README.md

```diff
@@ -5,23 +5,26 @@ This directory contains the docker file and useful scripts to help user to build
 Before doing this, make sure you have installed docker at your machine.
 
 ### Build the Spark Base Docker Image
-Go to folder docker/spark-centos and execute the following command to build Spark base docker image.
+Go to the directory [docker/spark-ubuntu](./docker/oap-ubuntu) and execute the following command to build Spark base docker image.
 ``` 
-docker build --tag spark-centos:1.2.0 .
+git clone -b <oap-tool-version> https://github.com/oap-project/oap-tools.git
+cd oap-tools/integrations/oap/kubernetes/docker/spark-ubuntu
+docker build --tag spark-ubuntu:1.3.0 .
 ``` 
 
 ### Build the OAP Docker Image
-Go to folder docker/oap-centos and execute the following command to build OAP docker image which is based on Spark base docker image.
+Go to folder [docker/spark-ubuntu](./docker/oap-ubuntu) and execute the following command to build OAP docker image which is based on Spark base docker image.
 ``` 
-docker build --tag oap-centos:1.2.0 .
+cd ../oap-ubuntu
+docker build --tag oap-ubuntu:1.3.0 .
 ``` 
 
 ## Run Spark/OAP on Kubernetes
-Before doing this, we assume you have setup Kubernetes enironment and it worked properly. All the tool scripts are under "spark" or "oap" folder. 
+Before doing this, we assume you have set up Kubernetes environment andKubernetes works properly. All the tool scripts are under [spark](./spark) or [oap](./oap) directory. 
```

Comment:
You'd better remove those references. Because it is difficult to be consistent in the context。 => (./spark）

The space symbol is missing for  "andKubernetes works"

Reply:
done